apollo-federation/src/connectors/models/problem_location.rs

```diff
@@ -0,0 +1,19 @@
+use serde::Deserialize;
+use serde::Serialize;
+
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq, Hash, PartialOrd, Ord)]
```

Comment:
```suggestion
#[derive(Debug, Clone, Copy, Serialize, Deserialize, PartialEq, Eq, Hash, PartialOrd, Ord)]
```
`Copy` should be used for tiny types (like this one, which will be like 1 byte) because it's actually more efficient to copy the value around than to take pointers to it.